src/enclave-agent/Cargo.toml

```
@@ -13,7 +13,7 @@ clap = "2.33.3"
 # logger module
 env_logger = "0.10.0"
 
-image-rs = { git = "https://github.com/confidential-containers/image-rs", default-features = false, rev = "v0.4.0" }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", features = ["enclave-cc"], default-features = false, rev = "cc626a9" }
```

The `features` addition here is causing the RATS-TLS build issues:

```suggestion
image-rs = { git = "https://github.com/confidential-containers/image-rs", default-features = false, rev = "cc626a9" }
```

---

Thanks, I've updated